lib/mosaic-database/src/main/java/org/eclipse/mosaic/lib/database/Database.java

```diff
@@ -563,7 +578,7 @@ public Node addNode(String nodeId, GeoPoint position, boolean isTrafficLight) {
         }
 
         public boolean nodeExists(String nodeId) {
-           return database.getNode(nodeId) != null;
+            return database.getNode(nodeId) != null;
```

Comment:
delete that space

Reply:
I think it's actually right (I just auto-formatted). Beforehand there were only three spaces for whatever reason, now there are four.

Reply:
you're totally right :)